package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Authors Haven Project Front End",
   "main": "index.js",
   "scripts": {
-    "start:dev": "webpack-dev-server --mode development --open --hot",
+    "start:dev": "webpack-dev-server --mode development --open --hot --https",
```

Comment:
Why did you enable https in dev mode?

Reply:
@adesege I can't remember exactly but it is needed by either Facebook/Google to work for authentication